public/app/features/variables/guard.test.ts

```diff
@@ -164,6 +165,7 @@ describe('type guards', () => {
   const variableFactsObj: Record<VariableType | ExtraVariableTypes, VariableFacts> = {
     query:      { variable: createQueryVariable(),      isMulti: true,  hasOptions: true,  hasCurrent: true },
     adhoc:      { variable: createAdhocVariable(),      isMulti: false, hasOptions: false, hasCurrent: false },
+    groupby:    { variable: createGroupByVariable(),    isMulti: true,  hasOptions: false, hasCurrent: false },
```

Comment:
yeah - because we include `GroupByVariable` in the union of `TypedVariableModel`s, it's present in `VariableType` and therefore needs a case adding here. we could explicitly exclude it, but it starts to get a bit messy. i kind of like the fact that once you add a new variable type you have to add a test for it here, etc.

Reply:
Kk, i understand it now. Ok, that's fair, even though this ain't used and will soon be removed.